test/groovy/com/sap/piper/versioning/DockerArtifactVersioningTest.groovy

```diff
@@ -20,7 +20,24 @@ class DockerArtifactVersioningTest extends BasePiperTest{
 
     String passedDir
 
-    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this, 'test/resources/versioning/DockerArtifactVersioning')
+    JenkinsReadFileRule readFileRule = new JenkinsReadFileRule(this,
+        ['Dockerfile': ('''|FROM rootImage:1.2.3
+                           |
+                           |USER root
+                           |
+                           |ENV TEST 2.3.4
+                           |''' as CharSequence).stripMargin(),
+         Dockerfile_registryPort : ('''|FROM registry:4444/path/image:1.2.3
```

Comment:
shouldn't this and the next keys be strings?

Reply:
Actually it does not make a difference. Unquoted keys are coerced into `java.lang.String`s. For the sake of consistency I changed it from unquoted keys to strings.